haproxy_exporter.go

```diff
@@ -102,6 +102,8 @@ var (
 		16: newServerMetric("redispatch_warnings_total", "Total of redispatch warnings.", nil),
 		17: newServerMetric("up", "Current health status of the server (1 = UP, 0 = DOWN).", nil),
 		18: newServerMetric("weight", "Current weight of the server.", nil),
+		21: newServerMetric("check_fail_total", "Total number of failed health checks.", nil),
+		24: newServerMetric("downtime_total", "Total downtime in seconds.", nil),
```

Comment:
We prefer including the unit in the metric name as well: `downtime_seconds_total` https://prometheus.io/docs/practices/naming/